README.md

```diff
@@ -22,31 +22,40 @@ additional installation and setup information for using the library with various
 
 ```xml
 <dependency>
-  <groupId>com.lightstep.tracer</groupId>
-  <artifactId>lightstep-tracer-jre</artifactId>
-  <version> VERSION </version>
+    <groupId>com.lightstep.tracer</groupId>
+    <artifactId>lightstep-tracer-jre</artifactId>
+    <version> VERSION </version>
 </dependency>
 
 <dependency>
-   <groupId>com.lightstep.tracer</groupId>
-   <artifactId>tracer-grpc</artifactId>
-   <version> VERSION </version>
+    <groupId>com.lightstep.tracer</groupId>
+    <artifactId>tracer-grpc</artifactId>
+    <version> VERSION </version>
 </dependency>
 
+<!-- Provided Dependencies -->
 <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-netty</artifactId>
-      <version>1.4.0</version>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-simple</artifactId>
+    <version>1.7.25</version>
+    <scope>provided</scope>
```

Comment:
I don't think we want `<scope>provided</scope>` here. We declare our dependencies as provided, but we expect someone who is including the our dependencies to do the providing. I might be wrong though.

Reply:
@MatthewDolan ping

Reply:
Maybe, I think it just isn't very clear. The expectation is that you do have these dependencies bound in your pom.xml. Can we chat IRL?